src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,99 @@
+package edu.kis.vh.nursery;
+
+/**
+ * Klasa {@code DefaultCountingOutRhymer} implementuje podstawową strukturę stosu
+ * o stałej pojemności. Umożliwia dodawanie, usuwanie i podglądanie elementów,
+ * a także sprawdzanie, czy struktura jest pusta lub pełna.
+ */
+public class DefaultCountingOutRhymer {
+
+    /**
+     * Maksymalna liczba elementów, które można przechować.
+     */
+    private static final int STACK_CAPACITY = 12;
+
+    /**
+     * Tablica przechowująca elementy stosu.
+     */
+    private final int[] numbers = new int[STACK_CAPACITY];
+
+    /**
+     * Wartość zwracana przy niepoprawnych operacjach.
+     */
+    private static final int RETURN_VAL = -1;
+
+    /**
+     * Indeks oznaczający pełny stos.
+     */
+    private static final int FULL = 11;
```

Comment:
This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).